site/components/pages/Resources/ResourcesList/styles.ts

```diff
@@ -46,3 +49,93 @@ export const searchInputSubmit = css`
     fill: var(--font-03);
   }
 `;
+
+export const showOnSmallScreens = css`
+  ${breakpoints.large} {
+    display: none;
+  }
+`;
+
+export const showOnBigScreens = css`
+  display: none;
+  ${breakpoints.large} {
+    display: block;
+  }
+`;
+
+export const sortbyContainer = css`
+  display: flex;
+  flex-direction: row;
+  gap: 1.2rem;
+  align-items: center;
+`;
+
+export const toggleMobileModalButton = css`
+  background-color: var(--surface-01);
+
+  svg {
+    fill: var(--font-01);
+  }
+
+  ${showOnSmallScreens}
+`;
+
+export const sortBySelectContainer = css`
+  display: none;
+
+  ${breakpoints.large} {
+    display: flex;
+    flex-direction: row;
+    gap: 1.2rem;
+    align-items: center;
+  }
+`;
+
+export const sortByButtons = css`
```

Comment:
on mobile these need justify-content: start and gap: 1.6rem

before fix:
![image](https://user-images.githubusercontent.com/88635679/196311218-5af2528a-f776-4810-a109-d298867b4f66.png)


after fix:

![image](https://user-images.githubusercontent.com/88635679/196311131-7296e6a2-087a-4898-b8ff-10e3e7accdf5.png)

Reply:
Done